cs_tools/cli/tools/git/config.py

```diff
@@ -0,0 +1,198 @@
+    ts = ctx.obj.thoughtspot
+
+    if org is not None:
+        ts.org.switch(org)
+
+    try:
+        r = ts.api_v2.vcs_git_config_update(
+            username=username,
+            access_token=access_token,
+            org_identifier=org,
+            branch_names=branch_names,
+            commit_branch_name=commit_branch_name,
+            enable_guid_mapping=enable_guid_mapping,
+            configuration_branch_name=configuration_branch_name
+        )
+
+        _show_configs_as_table([r.json()], title="Updated Configuration Details")
+
+    except HTTPStatusError as e:
+        rich_console.print(f"[bold red]Error creating the configuration: {e.response}.[/]")
+        rich_console.print(f"[bold red]{e.response.content}.[/]")
+
+
+@app.command(dependencies=[thoughtspot], name="search")
+def config_search(
+        ctx: typer.Context,
+        org: str = typer.Option(None, help="the org run in"),
+        org_ids: str = typer.Option(None, custom_type=MultipleChoiceType(),
+                                    help="The org IDs to get the configuration for")
+):
+    """
+    Searches for configurations.
+    """
+    ts = ctx.obj.thoughtspot
+
+    if org is not None:
+        ts.org.switch(org)
+
+    try:
+        r = ts.api_v2.vcs_git_config_search(
+            org_ids=org_ids
+        )
+
+        configs = r.json()
+        _show_configs_as_table(configs)
+
+    except HTTPStatusError as e:
+        rich_console.print(f"[bold red]Error creating the configuration: {e.response}.[/]")
+        rich_console.print(f"[bold red]{e.response.content}.[/]")
+
+
+@app.command(dependencies=[thoughtspot], name="delete")
+def config_delete(
+        ctx: typer.Context,
+        org: str = typer.Option(None, help="the org id to delete from"),
+        cluster_level: bool = typer.Option(False, help="the cluster level to use for the git repository"),
+):
+    """
+    Deletes a configuration for a cluster or org.
+    """
+    ts = ctx.obj.thoughtspot
+
+    if org is not None:
+        ts.org.switch(org)
+    else:
+        # DEV NOTE: delete doesn't take an org, so it will use whatever the last one was.
+        # It might be prudent to prompt to user if they want to continue.  It won't apply to
+        # non-org enabled clusters.
+        rich_console.print("[bold yellow]No org specified, the config in the current org will be deleted.[/]")
+
+    try:
+        r = ts.api_v2.vcs_git_config_delete(
+            cluster_level=cluster_level
+        )
+
+        rich_console.print(f"[green]Deleted the configuration: {r}.[/]")
+
+    except HTTPStatusError as e:
+        rich_console.print(f"[bold red]Error creating the configuration: {e.response}.[/]")
+        rich_console.print(f"[bold red]{e.response.content}.[/]")
+
+
+def _show_configs_as_table(configs: list[dict], title: str = "Configuration Details"):
+    """
+    Show the configurations as a table.
+    """
+    use_cnt = len(configs) > 1
+
+    cnt: int = 0
+    for _ in configs:
```

Comment:
`enumerate` gives you this incremental counting

```python
    use_count = len(configs) > 1

    for count, config in enumerate(configs, start=1):
        title = f"{title} {count}" if use_count else title
        table = Table(title=f"Configuration Details {count}", width=100)
        table.add_column("Property", width=25)
        table.add_column("Value", width=75)

        for k, v in config.items():
            table.add_row(k, str(v))

        rich_console.print(Align.center(table))
```

Reply:
Oh, that's good to know.  Svelte does this as well.  Changing.